webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.scss

```diff
@@ -0,0 +1,93 @@
+.ldap-wizard-wrapper {
+    background-color: rgba(var(--center-channel-color-rgb), 0.04);
+
+    .ldap-wizard-content-wrapper {
+        display: flex;
+        overflow: hidden; // Prevent double scrollbars on the body if content overflows
+        flex-grow: 1;
+        padding-top: 20px;
```

Comment:
I think this padding is incorrect since it causes the content to be cut off when you start scrolling. It should probably be part of the scrolling content and not the scroll container where it is now

<img width="859" alt="Screenshot 2025-05-23 at 5 11 22 PM" src="https://github.com/user-attachments/assets/cccd1845-b064-4b02-93f4-ad78c909b6e5" />

Reply:
Yep, good idea. Please check now @asaadmahmood -- I'm making some decision about the interactions.